Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -645,7 +644,7 @@ void GameCubePane::BrowseAGPRom(ExpansionInterface::Slot slot)
   ASSERT(ExpansionInterface::IsMemcardSlot(slot));
 
   QString filename = DolphinFileDialog::getSaveFileName(
-      this, tr("Choose a file to open"), QString::fromStdString(File::GetUserPath(D_GCUSER_IDX)),
+      this, tr("Choose a File to Open"), QString::fromStdString(File::GetUserPath(D_GCUSER_IDX)),
```

Comment:
This string and the first changed string in this file still have an indefinite article. Intentional?

Reply:
I feel like it fits better in this kind of longer phrase. I don't have any rigorous reasoning behind it.